### PR TITLE
Add edit stall function

### LINKED ISSUE #163: Add edit function for Stalls

### PR DESCRIPTION
**Pull request by MinHeinA**
Resolve #163

### CODE REVIEW

File: src/test/java/foodwhere/logic/commands/EditCommandTest.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import foodwhere.testutil.TypicalStalls;
 
 /**
- * Contains integration tests (interaction with the Model) and unit tests for EditCommand.
+ * Contains integration tests (interaction with the Model) and unit tests for SEditCommand.
  */
 public class EditCommandTest {
```
Comment (clarence-chew) [inline src/test/java/foodwhere/logic/commands/EditCommandTest.java:24]: Perhaps this should be renamed SEditCommandTest so we can split sedit and redit?

**Review by clarence-chew — COMMENTED** [review]
I have a question here, otherwise looks good to merge.

File: src/test/java/foodwhere/logic/commands/EditCommandTest.java (hunk 1 above)
Comment (MinHeinA) [reply src/test/java/foodwhere/logic/commands/EditCommandTest.java]: Fixed and resolved in the latest push. Thank you for spotting it.

**Review by Hikoya — APPROVED** [review]
LGTM.

**Pull request merged**